x-pack/test/functional/apps/code/explore_repository.ts

```diff
@@ -128,6 +128,16 @@ export default function exploreRepositoryFunctonalTests({
         });
         log.info('src-doc folder opened');
 
+        // Click breadcrumb does not affect file tree
```

Comment:
let's separate this test case into an independent one, otherwise, by adding test cases into the existing test will make it too complex. and could more prone to flaky test.